src/main/java/frc/robot/subsystems/shooter/ShooterSubsystem.java

```diff
@@ -187,30 +161,33 @@ public Command startShootingNoteCommand() {
 
   /**
    * Sets the elevation of the shooter all the way up
-   * @return
+   * @return Command
    */
   public Command setElevationUpCommand() {
     return Commands.runOnce(this::setElevatorUp);
   }
 
   /**
    * Sets the elevation of the shooter all the way down
-   * @return
+   * @return Command
    */
   public Command setElevationDownCommand() {
     return Commands.runOnce(this::setElevatorDown);
   }
 
   /**
    * Toggles the elevation of the shooter up/down
-   * @return
+   * @return Command
    */
   public Command toggleElevationCommand() {
     return Commands.runOnce(() -> {
-      if (m_elevationSolenoid.get() == Value.kForward) setElevatorDown(); else setElevatorUp();
+      if (shooterInputs.elevationSolenoid_state == Value.kForward) setElevatorDown(); else setElevatorUp();
```

Comment:
Writing an `if-else` statement like this, although _technically_ will not cause a problem, is not super readable, or maintainable if we need to change it in the future. Please revise this to be a multi-line if-else.